app/src/main/java/org/oppia/app/topic/info/TopicInfoFragment.kt

```diff
@@ -12,6 +12,18 @@ import javax.inject.Inject
 
 /** Fragment that contains info of Topic. */
 class TopicInfoFragment : InjectableFragment() {
+  companion object{
+    /** Returns a new [TopicInfoFragment]. */
+    fun newInstance(internalProfileId: Int, topicId: String): TopicInfoFragment{
```

Comment:
This function is actually not getting used anywhere. Checkout `ViewPagerAdapter` and see how can you use this function there. Also check if TopicInfoFragment is getting called from anywhere and if so, use `newInstance` accordingly.

Reply:
Done. `ViewPagerAdapter` seems to be the only file where changes were needed.

Reply:
Looks good. Thanks.